README.md

```diff
@@ -0,0 +1,3 @@
+# nla-pageview-job
```

Comment:
As a new person I'd be wondering:
- what is NLA?
- what's a NLA page view.

Something like really top level like
# NLA page view

Uploads Guardian page view data to NLA Media Access (NLA). 
NLA use this data to (do something?)

Reply:
Great, will try to find out more about what they use this data for :)